modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/src/main/java/com/liferay/headless/admin/workflow/internal/odata/entity/v1_0/WorkflowDefinitionEntityModel.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.headless.admin.workflow.internal.odata.entity.v1_0;
+
+import com.liferay.portal.kernel.search.Field;
+import com.liferay.portal.odata.entity.DateTimeEntityField;
+import com.liferay.portal.odata.entity.EntityField;
+import com.liferay.portal.odata.entity.EntityModel;
+import com.liferay.portal.odata.entity.StringEntityField;
+
+import java.util.Map;
+
+/**
+ * @author Nara Andrade
+ */
+public class WorkflowDefinitionEntityModel implements EntityModel {
+
+	public WorkflowDefinitionEntityModel() {
+		_entityFieldsMap = EntityModel.toEntityFieldsMap(
+			new DateTimeEntityField(
+				"dateModified",
+				locale -> Field.getSortableFieldName(Field.MODIFIED_DATE),
+				locale -> Field.MODIFIED_DATE),
+			new StringEntityField("name", locale -> Field.NAME));
```

Comment:
@naraandrade Please use this `new StringEntityField("name", locale -> Field.getSortableFieldName(Field.NAME)))` instead just to keep consistency since it is a sortable field.